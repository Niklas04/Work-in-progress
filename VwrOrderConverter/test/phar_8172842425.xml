<?xml version="1.0" encoding="ISO-8859-2" ?> <order  order_id="8172842425" order_cen="" date_cre="2018-08-22" provider_id="63073242" partner_id="19010290" branch_id="500000" compensation="0" text="">
<items>
<item qty="3.00" price="23.00" prod_id="710149" prod_title="Kádinka vysoká  150ml          153" code_pdk="4207149" code_apa="4207149" code_sukl="" code_ean="N"/>
<item qty="120.00" price="30.30" prod_id="712878" prod_title="Láhev na moè mu¾ská z PH  modré víèko 1l  VITRUM" code_pdk="2482843" code_apa="2482843" code_sukl="" code_ean="N"/>
<item qty="4.00" price="90.10" prod_id="713915" prod_title="Miska tøecí drsná pr. 8,1 cm 211a/1/0 " code_pdk="N" code_apa="" code_sukl="" code_ean="N"/>
<item qty="6.00" price="29.60" prod_id="716622" prod_title="Støièka UH 250 ml + uzávìr (skleò.kolínko)" code_pdk="N" code_apa="" code_sukl="" code_ean="N"/>
<item qty="8.00" price="43.20" prod_id="710944" prod_title="Tlouèek drsný 213a/1      115mm" code_pdk="8911423" code_apa="8911423" code_sukl="" code_ean="N"/>
<item qty="6.00" price="49.70" prod_id="710393" prod_title="Tlouèek drsný 213a/2     135mm" code_pdk="8911415" code_apa="8911415" code_sukl="" code_ean="N"/>
</items>
</order>
