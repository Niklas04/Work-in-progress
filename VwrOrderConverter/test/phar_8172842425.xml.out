BEGIN;8172842425;Pharmos, a.s.;VWR;ORDERS;2018-08-22;102511;
HEAD;8172842425;2018-08-22;;CZK;
SOLDTO;26002042;Pharmos, a.s.
SHIPTO;26005965;Pharmos, a.s. Brandýsek;;;;;Slánská 79;;;;;Brandýsek;273 41;CZ;marie.baladova@pharmos.cz;312 777 057
LINE;000001;VTRB632417012150;VTRB632417012150;Kádinka vysoká  150ml          153;3.000;;2018-08-22;23.00;69.00
LINE;000002;GAMA204811;GAMA204811;Láhev na moč mužská z PH  modré víčko 1l  VITRUM;120.000;;2018-08-22;30.30;3636.00
LINE;000003;JIZE211A/1/0;JIZE211A/1/0;Miska třecí drsná pr. 8,1 cm 211a/1/0 ;4.000;;2018-08-22;90.10;360.40
LINE;000004;TNOS632524325025;TNOS632524325025;Střička UH 250 ml + uzávěr (skleň.kolínko);6.000;;2018-08-22;29.60;177.60
LINE;000005;JIZE213A/1;JIZE213A/1;Tlouček drsný 213a/1      115mm;8.000;;2018-08-22;43.20;345.60
LINE;000006;JIZE213A/2;JIZE213A/2;Tlouček drsný 213a/2     135mm;6.000;;2018-08-22;49.70;298.20
END;8172842425
